infrastructure/terraform/components/reporting/s3_bucket_daily_report.tf

```diff
@@ -0,0 +1,51 @@
+resource "aws_s3_bucket" "daily_report" {
```

Comment:
Avoid the use of the term "daily_report" - the scope is broader than this. It's probably the "reporting" bucket.

Reply:
changed